### PR TITLE
tests: new tests for numeric eacl rules

### PR DESCRIPTION
**Pull request by evgeniiz321**

### CODE REVIEW

**Review by roman-khimov — COMMENTED** [review]
Can we also have some special eacls that touch system attributes like epoch, expiration and object size? We also want to check PUTs. Like: you can put objects of size less than X (2*big object size and try to put small, big and 3*big) and expiration less than Y.

**Comment by evgeniiz321** [comment]
@roman-khimov added missing scenarios.

File: pytest_tests/tests/acl/test_eacl_filters.py [hunk 1]
```diff
@@ -627,3 +657,556 @@ def test_extended_acl_allow_filters_object(
...
+
+        with allure.step(f"GET object with numeric value attribute should be allowed"):
+            assert can_get_object(
+                user_wallet.wallet_path,
+                cid,
+                oid,
+                file_path,
+                self.shell,
+                neofs_env=self.neofs_env,
+            ), f"GET is not allowed for this object, while it shouldn't be"
```
Comment (roman-khimov) [inline pytest_tests/tests/acl/test_eacl_filters.py:866]: should be?

File: pytest_tests/tests/acl/test_eacl_filters.py [hunk 2]
```diff
@@ -627,3 +657,556 @@ def test_extended_acl_allow_filters_object(
...
```
Comment (roman-khimov) [inline pytest_tests/tests/acl/test_eacl_filters.py:909]: And here.

File: pytest_tests/tests/acl/test_eacl_filters.py [hunk 3]
```diff
@@ -627,3 +657,556 @@ def test_extended_acl_allow_filters_object(
...
```
Comment (roman-khimov) [inline pytest_tests/tests/acl/test_eacl_filters.py:814]: should be

File: pytest_tests/tests/acl/test_eacl_filters.py [hunk 4]
```diff
@@ -627,3 +657,556 @@ def test_extended_acl_allow_filters_object(
...
+        with allure.step("Add test object to container"):
+            file_path = generate_file(complex_object_size)
+
+            oid = put_object_to_random_node(
+                user_wallet.wallet_path,
+                file_path,
+                cid,
+                shell=self.shell,
+                neofs_env=self.neofs_env,
+                lifetime=2,
```
Comment (roman-khimov) [inline pytest_tests/tests/acl/test_eacl_filters.py:993]: I think you should use `expire-at`, lifetime is relative to the current epoch which technically can change during the test.

File: pytest_tests/tests/acl/test_eacl_filters.py [hunk 5]
```diff
@@ -627,3 +657,556 @@ def test_extended_acl_allow_filters_object(
...
+                user_wallet.wallet_path,
+                cid,
+                shell=self.shell,
+                endpoint=self.neofs_env.sn_rpc,
+            )
+
+        small_file_path = generate_file(complex_object_size)
+
+        with allure.step(
+            f"PUT object with should not be allowed because size is LT {complex_object_size + 1}"
```
Comment (roman-khimov) [inline pytest_tests/tests/acl/test_eacl_filters.py:1071]: s/with/which/

File: pytest_tests/tests/acl/test_eacl_filters.py (hunk 1 above)
Comment (evgeniiz321) [reply pytest_tests/tests/acl/test_eacl_filters.py]: fixed

File: pytest_tests/tests/acl/test_eacl_filters.py (hunk 2 above)
Comment (evgeniiz321) [reply pytest_tests/tests/acl/test_eacl_filters.py]: fixed

File: pytest_tests/tests/acl/test_eacl_filters.py (hunk 3 above)
Comment (evgeniiz321) [reply pytest_tests/tests/acl/test_eacl_filters.py]: fixed

File: pytest_tests/tests/acl/test_eacl_filters.py (hunk 4 above)
Comment (evgeniiz321) [reply pytest_tests/tests/acl/test_eacl_filters.py]: fixed

File: pytest_tests/tests/acl/test_eacl_filters.py (hunk 5 above)
Comment (evgeniiz321) [reply pytest_tests/tests/acl/test_eacl_filters.py]: fixed

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**